README.md

```diff
@@ -43,7 +45,9 @@ Scalable Video Technology is licensed under the OSI-approved BSD+Patent license.
 # Documentation
 
 More details about the encoder usage can be found under:
--   [svt-hevc-encoder-user-guide](Docs/svt-hevc_encoder_user_guide.md)
+
+-   [SVT-HEVC Encoder User Guide](Docs/svt-hevc_encoder_user_guide.md)
+
```

Comment:
Please remove extra empty line.

Reply:
Removed empty line and pushed patch (Commit : 9db1593)